x-pack/test/api_integration/apis/synthetics/add_monitor.ts

```diff
@@ -169,7 +168,21 @@ export default function ({ getService }: FtrProviderContext) {
 
       const { body: apiResponse } = await addMonitorAPI(newMonitor);
 
-      epct(apiResponse).toEqual(epct.objectContaining({ max_attempts: maxAttempts }));
+      epct(apiResponse).toEqual(epct.objectContaining({ retest_on_failure: true }));
```

Comment:
Why remove the assertion regarding maxAttempts?

Reply:
because it's not returned anymore.

Reply:
All of these tests were originally written to support the internal use case. It should still return `max_attempts` which the UI is expect.

All of these test should therefore pass the appropriate param to make this as a UI request.